### PR TITLE
codeintel: move to scip-rust over lsif-rust docker image for auto-indexing

### PR DESCRIPTION
**Pull request by Strum355**
Requires https://github.com/sourcegraph/lsif-rust/pull/10 to land first

## Test plan

Nothing to test, just bumping image tags that have been tested prior

### CODE REVIEW

**Comment by tjdevries** [comment]
remove `lsif-rust` as an indexer type? or can we not do that?

**Comment by Strum355** [comment]
> remove `lsif-rust` as an indexer type? or can we not do that?

For some of the maps yea, for others we want to keep it as its used in the hrefs for the UI. Im waiting right now for docker hub to get a grip so I can update the shasums

File: enterprise/internal/codeintel/autoindexing/internal/inference/lang_rust_test.go [hunk 1]
```diff
@@ -23,8 +23,8 @@ func TestRustGenerator(t *testing.T) {
 					LocalSteps:  nil,
 					Root:        "",
 					Indexer:     expectedIndexerImage,
-					IndexerArgs: []string{"lsif-rust", "index"},
-					Outfile:     "dump.lsif",
+					IndexerArgs: []string{"scip-rust", "index"},
+					Outfile:     "dump.scip",
```
Comment (efritz) [inline enterprise/internal/codeintel/autoindexing/internal/inference/lang_rust_test.go:27]:
```suggestion
					Outfile:     "index.scip",
```

File: enterprise/internal/codeintel/autoindexing/internal/inference/lua/rust.lua [hunk 2]
```diff
@@ -2,7 +2,7 @@ local recognizer = require "sg.autoindex.recognizer"
 local pattern = require "sg.autoindex.patterns"
 
 local indexer = require("sg.autoindex.indexes").get "rust"
-local outfile = "dump.lsif"
+local outfile = "dump.scip"
```
Comment (efritz) [inline enterprise/internal/codeintel/autoindexing/internal/inference/lua/rust.lua:5]:
```suggestion
local outfile = "index.scip"
```

**Review by efritz — APPROVED** [review]

**Pull request merged**